course_contents/2_intro_to_python/notes/04. Destructuring.md

```diff
@@ -36,8 +34,13 @@ for index, element in enumerate(collection):
 ### The `zip()` function
 - the `zip()` function is used to combine two iterables into a single one
```

Comment:
Here maybe saying "collection" for the same reason as above?

Probably a visual explanation of `zip()` would work best.

For example:

Show the two lists side by side and then the zipped list underneath both of them, with an arrow coming from each of the top two lists into the corresponding elements.

Reply:
added the design notes, I will change it to collection if we agree on changing it everywhere 👍